### ISSUE TITLE
Error adding entities for domain water_heater with platform ge_home

### ISSUE THREAD

**Issue by garyak** [simbaja/ha_gehome#217]
HA update to 2024.1.0 with GE Home 0.6.8 results in these errors:
```
2024-01-04 06:32:30.473 ERROR (MainThread) [homeassistant.components.water_heater] Error adding entities for domain water_heater with platform ge_home
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 507, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 662, in _async_add_entity
    capabilities=entity.capability_attributes,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/water_heater/__init__.py", line 244, in capability_attributes
    if WaterHeaterEntityFeature.OPERATION_MODE in self.supported_features:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'int' is not iterable
2024-01-04 06:32:30.479 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 507, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 662, in _async_add_entity
    capabilities=entity.capability_attributes,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/water_heater/__init__.py", line 244, in capability_attributes
    if WaterHeaterEntityFeature.OPERATION_MODE in self.supported_features:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'int' is not iterable
```

**Comment by adamherbert** [comment]
Looks like this might be fixed in a Hass PR: https://github.com/home-assistant/core/pull/107091

It's tagged to be in the 2024.1.1 release.

**Comment by adamherbert** [comment]
I patched my system with the PR and it fixed the issue on my system.

**Comment by adamherbert** [comment]
This is fixed in HA 2024.1.1

**Comment by garyak** [comment]
Updated to 2024.1.1 resolved this issue.
